.github/workflows/ax.yml

```diff
@@ -52,27 +52,22 @@ jobs:
       CXX: ${{ matrix.compiler }}
     strategy:
       matrix:
-        image: ['2019-clang8', '2020-clang7', '2021-clang10']
```

Comment:
Think 2020-clang7 should stay here no? At least we still test against 2020 I think

Reply:
Well - it doesn't work because with these changes the minimum LLVM version becomes 8 so we can't use Clang 7? Unfortunately we don't have a '2020-clang8' we can build against.

Reply:
Hmm that's unfortunate... lets continue like this for now, I'll have to consider what to do here

Reply:
We can discuss adding a Clang 8 version to the 2020 container, but what I suggest might be simpler is to just accept LLVM 7 as the minimum version this year and then set LLVM 10 as the future minimum when we bump to CY2021. That's what I've implemented here for now.

Tagging @richhones.

Reply:
Hmm I guess testing against 2020 is considerably more important that dropping support for LLVM 7. I think this would be okay